src/response.js

```diff
@@ -10,7 +10,7 @@ export default function responseMw(settings) {
     return next => async options => {
         const result = await next();
 
-        if (isAPIError(result) || options[settings.key]) {
+        if (isAPIError(result) || (options && options[settings.key])) {
```

Comment:
It seems you made mistake here. Did you mean `!options || !(options && options[settings.key])`?

Reply:
It's my fault. You are right here.